src/components/inputs/ui-select-extended/ui-select-extended.tsx

```diff
@@ -57,13 +59,23 @@ const UISelectExtended: React.FC<OHRIFormFieldProps> = ({ question, handler, onC
   };
 
   const debouncedSearch = debounce((searchterm, dataSource) => {
+    setItems([]);
     setIsLoading(true);
     dataSource.fetchData(searchterm, config).then((dataItems) => {
       setItems(dataItems.map(dataSource.toUuidAndDisplay));
+
       setIsLoading(false);
     });
   }, 300);
 
+  const processSearchableValues = (value) => {
+    dataSource.fetchData(null, config, value).then((dataItem) => {
+      setSavedSearchableItem(dataItem);
+      setSavedSearchableItemDisplay(dataItem.display);
```

Comment:
Is there a reason why these 2 are separated? it looks like duplication

Reply:
Fixed